setup.py

```diff
@@ -0,0 +1,29 @@
+
+
+import os
+
+from setuptools import setup, find_packages
+
+from heroprotocol import __version__
+
+
+setup(
+    name='heroprotocol',
+    version=__version__,
+    author='Blizzard Entertainment',
+    author_email='???',
+    packages=find_packages(exclude=['tests']),
```

Comment:
Tests would help my port to Javascript greatly. See [Farof/heroprotocoljs](/Farof/heroprotocoljs). It works fine but not all code paths have been tested so I wouldn't be surprised at unforeseen errors popping up.